duo/pam/init.sls

```diff
@@ -13,7 +17,7 @@ pam_duo:
 
   file.managed:
     - name: {{ duo.config_dir }}pam_duo.conf
-    - source: salt://{{ slspath }}/files/duo.conf
+    - source: salt://duo/files/duo.conf
```

Comment:
Was there a reason to replace `{{ slspath }}` with the static 'duo'? Using slspath Jinja allowed the formula to be exported as any name without breaking the state.